apps/src/templates/sectionProgress/progressTables/ProgressTableStudentList.jsx

```diff
@@ -58,14 +57,9 @@ export default class ProgressTableStudentList extends React.Component {
       scriptData.name,
       rowData.student.id
     );
-    let fullName;
-    if (!!DCDO.get('family-name-features', false)) {
-      fullName = rowData.student.familyName
-        ? `${rowData.student.name} ${rowData.student.familyName}`
-        : rowData.student.name;
-    } else {
-      fullName = rowData.student.name;
-    }
+    let fullName = rowData.student.familyName
```

Comment:
if we don't modify fullName anywhere else, i think it can be a const now!

Reply:
🙌🏼🙌🏼🙌🏼